CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 1.7.0-beta.1
+
+## ✨ Features
+* Add a new constructor option to `mapboxgl.Popup`, `closeOnMove`, that closes the popup when the map's position changes ([#9163](https://github.com/mapbox/mapbox-gl-js/pull/9163))
+* Add third parameter to once event handler ([#8875](https://github.com/mapbox/mapbox-gl-js/pull/8875))
+* Allow programmatic construction of style even when one is not passed in initially ( h/t @stepankuzmin ) ([#8924](https://github.com/mapbox/mapbox-gl-js/pull/8924))
+* Add option to use a feature property as ID for feature state ([#8987](https://github.com/mapbox/mapbox-gl-js/pull/8987))
+* Use `browser.devicePixelRatio` instead of directly calling `window.devicePixelRatio` ([#9063](https://github.com/mapbox/mapbox-gl-js/pull/9063))
```

Comment:
This isn't really a feature , right?

Reply:
I think it is because it introduces the ability to:

1. show map inside scaled element (transform: scale) with working mouse events
2. change map rendering pixel ratio.

1 could be considered a bug but is featurey to me, I think 2 is definitely a feature.

Reply:
You can add "... to be able to override it" to signify that it's a feature.

Reply:
Done.